### ISSUE TITLE
Service integration connect button not appearing on Linux

### ISSUE THREAD

**Issue by LiamDawe** [obsproject/obs-studio#4798]
### Operating System Info

Other

### Other OS

Arch Linux

### OBS Studio Version

27.0.0

### OBS Studio Log URL

https://gist.github.com/LiamDawe/3355b8ef3ef4a4a3c303b59728dde0cd

### Expected Behavior

Go into settings, see the Connect Account (as [announced](https://twitter.com/OBSProject/status/1399851168000405504)) button to login with Twitch.

### Current Behavior

There is no Connect Account button.

### Steps to Reproduce

Compile a standalone Linux build as per the instructions.

Try to find the Connect Account button.

Cry?

### Anything else we should know?

Browser docks work fine, so the browser stuff is there but no Connect Account option exists on Linux for me.

**Comment by dodgepong** [comment]
Which service are you trying to connect to? The "Connect Account" button should only appear for Twitch and Restream at this time.

**Comment by LiamDawe** [comment]
Twitch, also tried Restream, it doesn't appear.

**Comment by denisschultz** [comment]
Not on Arch - it is only for Ubuntu.

**Comment by tytan652** [comment]
Service integration is not available on ArchLinux package even AUR recipe

**Comment by LiamDawe** [comment]
Is there a reason for that? So not even the Snap package either, ONLY the Ubuntu PPA?!

**Comment by mihawk90** [comment]
> Is there a reason for that?

The connections require API keys/secrets that are not readily available.

**Comment by tytan652** [comment]
PPA packages are official packages made by OBS  team.

The needed Client ID and Secrets for Twitch and Restream is only owned by the team.

Other packagers should do without it.

**Comment by LiamDawe** [comment]
That's frustrating, there must be a way around that somehow? Perhaps with an AppImage that could have it all bundled in?

**Comment by mihawk90** [comment]
All you can do in the service integration you can do with custom browser docks. It's is just a convenience thing.

Maybe some day, but not currently, no.

**Comment by LiamDawe** [comment]
Well, thanks for explaining carefully for me anyway! Appreciated.

**Comment by rashdanml** [comment]
> > Anything else we should know?
> Browser docks work fine, so the browser stuff is there but no Connect Account option exists on Linux for me.

Am I missing something? Just installed obs-studio through pacman, and I don't see this option in View -> Docks

**Comment by Corben78** [comment]
As any user on those platforms can register apps with their apis, would it be possible to enable the user to provide those keys to get the service login integration? E.g. via environment variables?

**Comment by Gol-D-Ace** [comment]
> As any user on those platforms can register apps with their apis, would it be possible to enable the user to provide those keys to get the service login integration? E.g. via environment variables?

We additionally now offer a Flatpak which does include this feature and runs on multiple Distributions.

https://flathub.org/apps/details/com.obsproject.Studio

**Comment by v-127** [comment]
I 

> > As any user on those platforms can register apps with their apis, would it be possible to enable the user to provide those keys to get the service login integration? E.g. via environment variables?
> 
> We additionally now offer a Flatpak which does include this feature and runs on multiple Distributions.
> 
> https://flathub.org/apps/details/com.obsproject.Studio

I don't know if this already changed, but it just doesn't work.
![image](https://github.com/obsproject/obs-studio/assets/104890279/7c93bc6c-eb03-4713-b416-226c6f2aa992)

**Comment by mihawk90** [comment]
> I don't know if this already changed, but it just doesn't work.

Flatpak has had Logins for quite some time.
In all likelyhood you are on Wayland and some of the browser features are disabled on Wayland due to obsproject/obs-browser#279

On a different note: Posting in a n issue that has been closed 3 years ago is rarely going to work out. I just happened to have this still subscribed for some reason. GitHub is also not used for support requests.
